src/AbstractContainer.php

```diff
@@ -157,6 +157,10 @@ protected function setManager(?Manager $manager = null)
      */
     public function getManager()
     {
+        if (null === $this->manager) {
```

Comment:
Is it possible to have a test case for this? :|

Reply:
not sure, as I stated before, it probably a phpunit bug, as after it applied, it got different error when I tried manually at https://github.com/laminas/laminas-mvc-plugin-fileprg/pull/18

```bash
1) LaminasTest\Mvc\Plugin\FilePrg\FilePrgDataMergingTest::testCorrectInputDataMerging
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
                 'name' => 'test.jpg'
                 'type' => 'image/jpeg'
                 'size' => 20480
-                'tmp_name' => '/Users/samsonasik/www/oss-contribute/laminas-mvc-plugin-fileprg/test/TestAsset/testfile.jpg'
+                'tmp_name' => '/Users/samsonasik/www/oss-contribute/laminas-mvc-plugin-fileprg/test/TestAsset/nullfile_copy'
```